app/views/account_links/_form.html.slim

```diff
@@ -15,6 +15,9 @@
       .input-group
         = f.text_field :api_key, data: {toggle: 'tooltip', placement: 'bottom'}, title: t('.info.token'), class: 'form-control'
         = button_tag t('.button.generate'), type: 'button', class: 'generate-api-key-token btn btn-light'
+    .field-element.form-group
+      = f.label :proforma_version, AccountLink.human_attribute_name('proforma_version'), class: 'form-label'
+      = f.select :proforma_version, options_for_select(ProformaXML::SCHEMA_VERSIONS, @account_link.proforma_version), {include_blank: t('account_links.common.proforma_version.select_default', latest_version: ProformaXML::SCHEMA_VERSION_LATEST)}, {class: 'form-select'}
```

Comment:
Do we have any consistent use of select2 vs native selects? I was shortly wondering whether we should use select2 here, too. However, the only other place we have a single select is for a task's license, where we don't enable select2 either. 

The only place where it is used consistently is for the advanced search filter, where also single selects have it enabled.

Reply:
I try to use normal selects where possible and select2 if I want a search box or I want to use another feature of select2

Reply:
Okay, makes sense. For now, I would say that's fine :+1:.